apis/apps/v1alpha1/type.go

```diff
@@ -1057,6 +1048,31 @@ type ComponentVars struct {
 	PodFQDNs *VarOption `json:"podFQDNs,omitempty"`
 }
 
+// ClusterVarSelector selects a var from a Cluster.
+type ClusterVarSelector struct {
+	// The Component to select from.
+	ClusterObjectReference `json:",inline"`
```

Comment:
is ClusterObjectReference not needed at all? The definition of ClusterObjectReference is to locate the referenced object inside the same Cluster.

Reply:
Removed